src/state/state-persister.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const mobTimerDir = path.join(os.homedir(), '.mob-timer')
+const stateFile = path.join(mobTimerDir, 'state.json')
+const oldStateFile = path.join(os.tmpdir(), 'state.json')
+
+function read() {
+    if (fs.existsSync(stateFile)) {
```

Comment:
I like the removal of the `init` I had originally conceptualized. I can appreciate the desire for eliminating temporal coupling.